### ISSUE TITLE
Bad frames, unusable depth data

### ISSUE THREAD

**Issue by kovacsgabor1** [OnionIoT/tau-lidar-camera#20]
### Python Library versions
> Run `pip list | grep TauLidar` in the command-line and paste the output. Include some details on your system as well

TauLidarCamera 0.0.5
TauLidarCommon 0.0.2
TauLidarServer 0.0.7

### Camera Info
> If possible, paste the output of `camera.info()` from Python (after camera has been successfully opened)

    model:      4.0
    firmware:   3.3
    uid:        970189
    resolution: 160x60
    port:       /dev/cu.usbmodem00000000001A1

### Describe the bug

The camera gives bad or corrupted images.

<img width="636" alt="Screenshot 2022-02-01 at 23 35 58" src="https://user-images.githubusercontent.com/57663761/152058613-d5b079f4-dfef-4fb7-914a-1054ebabd3dc.png">

### Steps to Reproduce

After the issue, the camera never gives correct images.

### What is expected?

Correct depth image.

### What is actually happening?

I was using the LidarCamera few times with original Apple USB-C to USB-C cable with M1 MacBook Pro, it worked flawlessly, but at a random time it gives me corrupted depth image. After restarting the camera it remains the same. I tried updating all the softwares but it didn't helped.

### Any additional comments?

The camera hardware looks like its working normally. 

<img width="655" alt="Screenshot 2022-02-02 at 00 12 11" src="https://user-images.githubusercontent.com/57663761/152060208-60caf33f-4dc9-4183-a721-c4774e537e03.png">

**Comment by dhruvsheth-ai** [comment]
@greenbreakfast Facing the same issue. Is Onion planning on any revamps to the software? This repo looks almost dead.

**Comment by kovacsgabor1** [comment]
I don't think so. The bug remains after a few hours of debugging and trying, without any kind of success.

**Comment by greenbreakfast** [comment]
what sort of output do you get when you run any of the [example programs](https://github.com/OnionIoT/tau-lidar-camera/tree/master/examples)?

Specifically interested in the [checkCamera program](https://github.com/OnionIoT/tau-lidar-camera/tree/master/examples#checkcamerapy-program) output
